index.html

```diff
@@ -21,6 +21,14 @@
                 </ul>
             </nav>
         </div>
+        <div class="nav-menu">
+          <button id="close-btn" type="button"><img src="images/Close-Icon.png" alt="close-btn"></button>
+          <ul>
+            <li><a class="nav-link" href="#works">Portfolio</a></li>
+            <li><a class="nav-link" href="#about">About</a></li>
+            <li><a class="nav-link" href="#contact">Contact</a></li>
+          </ul>
+        </div>
```

Comment:
- [ ] It would be better for the background not to be scrollable when the mobile nav is open, this will create a better user experience  👍🏿 


- [ ] Kindly make sure that on desktop the mobile menu is not visible, you can use a the `display:none`  property to hide elements on your page 👍🏿  

![Screenshot_84](https://github.com/BenMKT/myPortfolio/assets/69638013/4ac7b1f1-61d1-4c06-9285-51b107074bef)

Reply:
- Add position fixed to make mobile `nav-menu` fixed on the screen to resolve the review bug.
- Already added `display: none` property in the desktop media query hence mobile menu is not visible in the desktop version.